### PR TITLE
feat: pullToRefresh Appium tests

### PR DESCRIPTION
**Pull request by igorgiumellizup**
### Related Issues

#1626 

### Description and Example

Appium PullToRefresh feature 

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.automatedtests.builders
+
+import br.com.zup.beagle.automatedtests.constants.LOGO_BEAGLE
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:19]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/63de25a1aefd4dc376f6e61451d92cfe545a0593/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L19"></span>The import 'br.com.zup.beagle.automatedtests.constants.LOGO_BEAGLE' is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import br.com.zup.beagle.ext.unitReal
+import br.com.zup.beagle.widget.action.Alert
+import br.com.zup.beagle.widget.action.SetContext
+import br.com.zup.beagle.widget.context.ContextData
+import br.com.zup.beagle.widget.context.expressionOf
+import br.com.zup.beagle.widget.core.ScrollAxis
+import br.com.zup.beagle.widget.core.Size
+import br.com.zup.beagle.widget.layout.Container
+import br.com.zup.beagle.widget.layout.NavigationBar
+import br.com.zup.beagle.widget.layout.NavigationBarItem
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:31]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/63de25a1aefd4dc376f6e61451d92cfe545a0593/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L31"></span>The import 'br.com.zup.beagle.widget.layout.NavigationBarItem' is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import br.com.zup.beagle.widget.layout.Screen
+import br.com.zup.beagle.widget.layout.ScrollView
+import br.com.zup.beagle.widget.ui.Image
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:34]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/63de25a1aefd4dc376f6e61451d92cfe545a0593/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L34"></span>The import 'br.com.zup.beagle.widget.ui.Image' is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import br.com.zup.beagle.widget.ui.ImagePath
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:35]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/63de25a1aefd4dc376f6e61451d92cfe545a0593/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L35"></span>The import 'br.com.zup.beagle.widget.ui.ImagePath' is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.automatedtests.builders
+
+import br.com.zup.beagle.core.Style
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:19]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/c09330fde7e5ba8cacb027a2ef9399a25f845851/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L19"></span>The import 'br.com.zup.beagle.automatedtests.constants.LOGO_BEAGLE' is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+import br.com.zup.beagle.widget.action.Alert
+import br.com.zup.beagle.widget.action.SetContext
+import br.com.zup.beagle.widget.context.ContextData
+import br.com.zup.beagle.widget.context.expressionOf
+import br.com.zup.beagle.widget.core.ScrollAxis
+import br.com.zup.beagle.widget.core.Size
+import br.com.zup.beagle.widget.layout.Container
+import br.com.zup.beagle.widget.layout.NavigationBar
+import br.com.zup.beagle.widget.layout.Screen
+import br.com.zup.beagle.widget.layout.ScrollView
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:31]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/c09330fde7e5ba8cacb027a2ef9399a25f845851/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L31"></span>The import 'br.com.zup.beagle.widget.layout.NavigationBarItem' is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+import br.com.zup.beagle.widget.ui.PullToRefresh
+import br.com.zup.beagle.widget.ui.Text
+
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:34]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/c09330fde7e5ba8cacb027a2ef9399a25f845851/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L34"></span>The import 'br.com.zup.beagle.widget.ui.Image' is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+object PullToRefreshScreenBuilder {
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt:35]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/c09330fde7e5ba8cacb027a2ef9399a25f845851/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/PullToRefreshScreenBuilder.kt#L35"></span>The import 'br.com.zup.beagle.widget.ui.ImagePath' is unused.</td>
    </tr>
  </tbody>
</table>

**Review by paulomeurerzup — DISMISSED** [review]

**Review by paulomeurerzup — APPROVED** [review]

**Review by viniciusguardieirozup — APPROVED** [review]

**Pull request merged**